main/src/ca/uwaterloo/flix/language/phase/LambdaLift.scala

```diff
@@ -376,9 +375,14 @@ object LambdaLift {
     }
 
     def visitJvmMethod(method: SimplifiedAst.JvmMethod): LiftedAst.JvmMethod = method match {
-      case SimplifiedAst.JvmMethod(ident, fparams0, _, retTpe, purity, loc) =>
+      case SimplifiedAst.JvmMethod(ident, fparams0, exp, retTpe, purity, loc) =>
         val fparams = fparams0 map visitFormalParam
-        LiftedAst.JvmMethod(ident, fparams, retTpe, purity, loc)
+        visitExp(exp) match {
+          case closure: LiftedAst.Expression.Closure => 
```

Comment:
This pattern match should no longer be required right? Now you can just use the result of visitExp directly?